x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/deprecation_item.tsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { FunctionComponent } from 'react';
+import {
+  EuiAccordion,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiText,
+  EuiCallOut,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import type { DomainDeprecationDetails } from 'kibana/public';
+import { DeprecationHealth } from '../shared';
+import { LEVEL_MAP } from '../constants';
+import { StepsModalContent } from './steps_modal';
+
+const i18nTexts = {
+  getDeprecationTitle: (domainId: string) => {
+    return i18n.translate('xpack.upgradeAssistant.deprecationGroupItemTitle', {
+      defaultMessage: "'{domainId}' is using a deprecated feature",
+      values: {
+        domainId,
+      },
+    });
+  },
+  docLinkText: i18n.translate('xpack.upgradeAssistant.deprecationGroupItem.docLinkText', {
+    defaultMessage: 'View documentation',
```

Comment:
I think this may have been carried over from the ES deprecations list, where there may be one deprecation type that affects several indices, and therefore we only link to the docs once. @dborodyansky is doing an audit of the ES deprecations page vs. Kibana and making sure both UIs are aligned. I'm going to hold off on making any changes here until then.